### ISSUE TITLE
available details not contained in the table

### ISSUE THREAD

**Issue by IndrajeetPatil** [sfirke/packagemetrics#52]
For example, here `last_issue_closed`, `contributors`, etc. columns are empty, although these details are available.

Additionally, `depends_count` is incorrect for both packages.

``` r
library(packagemetrics)

ggstats_and_ggpubr <- package_list_metrics(c("ggstatsplot", "ggpubr"))

metrics_table(ggstats_and_ggpubr)
```

<table class="table table-condensed">

<thead>

<tr>

<th style="text-align:right;">

package

</th>

<th style="text-align:right;">

published

</th>

<th style="text-align:right;">

dl\_last\_month

</th>

<th style="text-align:right;">

stars

</th>

<th style="text-align:right;">

tidyverse\_happy

</th>

<th style="text-align:right;">

has\_tests

</th>

<th style="text-align:right;">

vignette

</th>

<th style="text-align:right;">

last\_commit

</th>

<th style="text-align:right;">

last\_issue\_closed

</th>

<th style="text-align:right;">

contributors

</th>

<th style="text-align:right;">

depends\_count

</th>

<th style="text-align:right;">

reverse\_count

</th>

</tr>

</thead>

<tbody>

<tr>

<td style="text-align:right;">

<span style="font-weight: bold">ggstatsplot</span>

</td>

<td style="text-align:right;">

2020-10-31

</td>

<td style="text-align:right;">

<span style="display: inline-block; direction: rtl; border-radius: 4px; padding-right: 2px; background-color: #56A33E; width: 2.94%">6781</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #56a33e">999</span>

</td>

<td style="text-align:right;">

<span style="color: purple">
<i class="glyphicon glyphicon-glass"></i>
</span>

</td>

<td style="text-align:right;">

<span style="color: green">
<i class="glyphicon glyphicon-ok"></i>
</span>

</td>

<td style="text-align:right;">

<span style="color: green">
<i class="glyphicon glyphicon-ok"></i>
</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #f06b13">0.0</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #f06b13"></span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #ffffff"></span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #ffffff">1</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #ffffff">0</span>

</td>

</tr>

<tr>

<td style="text-align:right;">

<span style="font-weight: bold">ggpubr </span>

</td>

<td style="text-align:right;">

2020-06-27

</td>

<td style="text-align:right;">

<span style="display: inline-block; direction: rtl; border-radius: 4px; padding-right: 2px; background-color: #56A33E; width: 100.00%">230781</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #ffffff">667</span>

</td>

<td style="text-align:right;">

<span style="color: purple">
<i class="glyphicon glyphicon-glass"></i>
</span>

</td>

<td style="text-align:right;">

<span style="color: red">
<i class="glyphicon glyphicon-remove"></i>
</span>

</td>

<td style="text-align:right;">

<span style="color: white">
<i class="glyphicon glyphicon-ok"></i>
</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #ffffff">1.3</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #f06b13"></span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #ffffff"></span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #1cc2e3">2</span>

</td>

<td style="text-align:right;">

<span style="display: block; padding: 0 4px; border-radius: 4px; background-color: #1cc2e3">74</span>

</td>

</tr>

</tbody>

</table>

<sup>Created on 2020-11-27 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by jsta** [comment]
The `depends_count` numbers look OK to me. `ggstatsplot` only depends on `"R (≥ 3.6.0)"` and `ggpubr` depends on `"R (>= 3.1.0), ggplot2"`.

**Comment by jsta** [comment]
I can confirm that pulling the Github fields `last_issue_closed` and `contributors` appears to be broken. Not just for these two example packages.

**Comment by sfirke** [comment]
closed in #53 by @jsta
